account_invoice_base_invoicing_mode/models/account_invoice.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+from odoo.addons.queue_job.job import job, related_action
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @job(default_channel="root.invoice_validation")
```

Comment:
Isn't the @job decorator deprecated?
https://github.com/OCA/queue/pull/274

Reply:
Yes it is, but it still works for v13.
It will be changed on the v14 migration.